scss/common/_forms.scss

```diff
@@ -58,6 +58,7 @@
         border-style: solid;
         box-sizing: border-box;
         position: relative;
+        transition: all .1s ease; // sass-lint:disable-block no-transition-all
```

Comment:
`border-color` and `box-shadow` should also be subject to transition, as they also change through different states.